src/IssueViz.Security/OpenInIDE/ListenerFactory.cs

```diff
@@ -28,11 +30,11 @@ namespace SonarLint.VisualStudio.IssueVisualization.Security.OpenInIDE
     internal interface IListenerFactory
     {
         /// <summary>
-        /// Attempts to create and return a new HTTP listener listening
+        /// Attempts to create and return a new HTTP listener listening(
```

Comment:
I've made the minimum changes required to use the OWIN classes that wrap the HttpListener so it will be easier to test the rest of the implementation. However, these changes don't make the existing factory class any more testable.